### DISCUSSION TITLE
Running recurrence job in all background servers?

### DISCUSSION THREAD

**Discussion by ap-sree** [jobrunr/jobrunr#821] [General]
I'm exploring few scheduler libs to check potential fit for my requirement. I understand jobrunr has optimistic lock for running the jobs. Is there any way to run the reoccurrence job in all background servers.

**Comment by rdehuyss** [comment]
Please see https://github.com/jobrunr/jobrunr/discussions/826.

**Reply by ap-sree** [reply]
Thanks for the reply
